src/cpp/src/continuous_batching/pipeline_impl.hpp

```diff
@@ -135,7 +135,8 @@ class ContinuousBatchingPipeline::ContinuousBatchingImpl : public ContinuousBatc
     generate(const std::vector<ov::Tensor>& input_ids,
              const std::vector<GenerationConfig>& sampling_params,
              const StreamerVariant& streamer,
-             std::optional<std::vector<ov::Tensor>> token_type_ids = std::nullopt) override;
+             const std::optional<std::vector<ov::Tensor>> token_type_ids = std::nullopt,
+             const std::optional<std::vector<std::pair<ov::Tensor, std::optional<int64_t>>>> position_ids = std::nullopt) override;
```

Comment:
How about `position_ids_list` or `position_ids_vector` not to be confused with single `position_ids` item/element?

Reply:
Yes, renamed to `position_ids_list`.